draft-bbs-signatures.md

```diff
@@ -1705,6 +1715,21 @@ Let the prover be in possession of a BBS signature `(A, e, s)` with `A = B * (1/
  <seriesInfo name="Springer," value="Cham"/>
 </reference>
 
+<reference anchor="SGS" target="https://link.springer.com/chapter/10.1007/978-3-540-28628-8_3">
```

Comment:
```suggestion
<reference anchor="BBS04" target="https://link.springer.com/chapter/10.1007/978-3-540-28628-8_3">
```
Because of my comment above (citing using the authors initials).